gitlab/v4/objects.py

```diff
@@ -1940,6 +1942,12 @@ def create(self, data, **kwargs):
         return CreateMixin.create(self, data, path=path, **kwargs)
 
 
+class ProjectPipelineJobManager(RetrieveMixin, RESTManager):
+    _path = '/projects/%(project_id)s/pipelines/%(pipeline_id)s/jobs'
+    _obj_cls = ProjectJob
```

Comment:
I'd rather use a new `ProjectPipelineJob` instead of `ProjectJob`. This is choice I made for other objects and managers, so I'd rather stick to this logic.